### PR TITLE
planner: update documents about cost model

### PR DESCRIPTION
**Pull request by qw4990**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -840,12 +840,11 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
-> - 当前 Cost Model Version 2 为实验特性，不建议在生产环境中使用。
 > - 切换代价模型版本可能会引起查询计划的变动。
 
 - 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
-- 默认值：`1`
+- 默认值：`2`
 - 可选值：`[1, 2]`
 - TiDB v6.2.0 引入了代价模型 [Cost Model Version 2](/cost-model.md#cost-model-version-2)，在内部测试中比此前版本的代价模型更加准确。
 - 通过将 `tidb_cost_model_version` 设置为 `2` 可以启用 Cost Model Version 2 代价模型，设置为 `1` 则继续使用 Cost Model Version 1 代价模型。
```
Comment (Oreoxmt) [inline system-variables.md:850]:
```suggestion
- 通过将 `tidb_cost_model_version` 设置为 `2` 可以使用 Cost Model Version 2 代价模型，设置为 `1` 则使用 Cost Model Version 1 代价模型。
```

File: cost-model.md [hunk 2]
```diff
@@ -40,13 +40,10 @@ mysql> SHOW CREATE TABLE t;
...
-
 TiDB v6.2.0 引入了新的代价模型 Cost Model Version 2。
 
 Cost Model Version 2 对代价公式进行了更精确的回归校准，调整了部分代价公式，比此前版本的代价公式更加准确。
 
-你可以通过设置变量 [`tidb_cost_model_version`](/system-variables.md#tidb_cost_model_version-从-v620-版本开始引入) 来控制代价模型的版本。
\ No newline at end of file
+你可以通过设置变量 [`tidb_cost_model_version`](/system-variables.md#tidb_cost_model_version-从-v620-版本开始引入) 来控制代价模型的版本。
+
+需要注意切换 Cost Model 版本可能引起执行计划的变动。
```
Comment (Oreoxmt) [inline cost-model.md:49]:
```suggestion
需要注意切换代价模型版本可能引起执行计划的变动。
```

File: system-variables.md (hunk 1 above)
Comment (Oreoxmt) [inline system-variables.md:850]:
```suggestion
- 可选值：
    - `1`：使用 Cost Model Version 1 代价模型。
    - `2`：使用 Cost Model Version 2 代价模型。TiDB v6.2.0 引入了代价模型 [Cost Model Version 2](/cost-model.md#cost-model-version-2)，在内部测试中比此前版本的代价模型更加准确。
```

File: system-variables.md [hunk 3]
```diff
@@ -840,12 +840,11 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 > **警告：**
 >
-> - 当前 Cost Model Version 2 为实验特性，不建议在生产环境中使用。
 > - 切换代价模型版本可能会引起查询计划的变动。
```
Comment (Oreoxmt) [inline system-variables.md:843]:
```suggestion
> 切换代价模型版本可能会引起查询计划的变动。
```

**Review by time-and-fate — APPROVED** [review]

**Review by chrysan — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: system-variables.md [hunk 4]
```diff
@@ -840,15 +840,14 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
 - 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
-- 默认值：`1`
-- 可选值：`[1, 2]`
-- TiDB v6.2.0 引入了代价模型 [Cost Model Version 2](/cost-model.md#cost-model-version-2)，在内部测试中比此前版本的代价模型更加准确。
-- 通过将 `tidb_cost_model_version` 设置为 `2` 可以启用 Cost Model Version 2 代价模型，设置为 `1` 则继续使用 Cost Model Version 1 代价模型。
+- 默认值：`2`
+- 可选值：
+    - `1`：使用 Cost Model Version 1 代价模型。
+    - `2`：使用 Cost Model Version 2 代价模型。TiDB v6.2.0 引入了代价模型 [Cost Model Version 2](/cost-model.md#cost-model-version-2)，在内部测试中比此前版本的代价模型更加准确。
```
Comment (Oreoxmt) [inline system-variables.md:850]:
```suggestion
    - `2`：使用 Cost Model Version 2 代价模型。TiDB v6.2.0 引入了代价模型 [Cost Model Version 2](/cost-model.md#cost-model-version-2)，在内部测试中比 Version 1 版本的代价模型更加准确。
```

File: cost-model.md [hunk 5]
```diff
@@ -40,13 +40,10 @@ mysql> SHOW CREATE TABLE t;
...
-
 TiDB v6.2.0 引入了新的代价模型 Cost Model Version 2。
 
 Cost Model Version 2 对代价公式进行了更精确的回归校准，调整了部分代价公式，比此前版本的代价公式更加准确。
 
-你可以通过设置变量 [`tidb_cost_model_version`](/system-variables.md#tidb_cost_model_version-从-v620-版本开始引入) 来控制代价模型的版本。
\ No newline at end of file
+你可以通过设置变量 [`tidb_cost_model_version`](/system-variables.md#tidb_cost_model_version-从-v620-版本开始引入) 来控制代价模型的版本。
+
+需要注意切换代价模型版本可能引起执行计划的变动。
```
Comment (shichun-0415) [inline cost-model.md:49]:
```suggestion
需要注意，切换代价模型版本可能引起执行计划的变动。
```

**Review by shichun-0415 — APPROVED** [review]

File: system-variables.md [hunk 6]
```diff
@@ -840,15 +840,14 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
 
 - 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
-- 默认值：`1`
-- 可选值：`[1, 2]`
-- TiDB v6.2.0 引入了代价模型 [Cost Model Version 2](/cost-model.md#cost-model-version-2)，在内部测试中比此前版本的代价模型更加准确。
-- 通过将 `tidb_cost_model_version` 设置为 `2` 可以启用 Cost Model Version 2 代价模型，设置为 `1` 则继续使用 Cost Model Version 1 代价模型。
+- 默认值：`2`
+- 可选值：
+    - `1`：使用 Cost Model Version 1 代价模型。
```
Comment (shichun-0415) [inline system-variables.md:849]: Seems we don't have any description of Cost Model Version 1 in cost-model.md. Should we add a description for it and tell the user when to select Version 1 and when to select Version 2?

Comment (Oreoxmt) [reply system-variables.md:849]: Version 2 is more accurate than Version 1. IMO, it is recommended to use Version 2 in v6.4.0 and later versions.

Comment (Oreoxmt) [inline system-variables.md:849]:
```suggestion
    - `1`：使用 Cost Model Version 1 代价模型。TiDB v6.3.0 及之前版本默认使用 Cost Model Version 1。
```

File: system-variables.md [hunk 7]
```diff
@@ -841,15 +841,14 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
 
 - 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
-- 默认值：`1`
-- 可选值：`[1, 2]`
-- TiDB v6.2.0 引入了代价模型 [Cost Model Version 2](/cost-model.md#cost-model-version-2)，在内部测试中比此前版本的代价模型更加准确。
-- 通过将 `tidb_cost_model_version` 设置为 `2` 可以启用 Cost Model Version 2 代价模型，设置为 `1` 则继续使用 Cost Model Version 1 代价模型。
+- 默认值：`2`
+- 可选值：
+    - `1`：使用 Cost Model Version 1 代价模型。TiDB v6.4.0 及之前版本默认使用 Cost Model Version 1。
```
Comment (TomShawn) [inline system-variables.md:850]:
```suggestion
    - `1`：使用 Cost Model Version 1 代价模型。TiDB v6.4.0 及之前的版本默认使用 Cost Model Version 1。
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

File: system-variables.md [hunk 8]
```diff
@@ -841,15 +841,14 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 > **警告：**
 >
-> - 当前 Cost Model Version 2 为实验特性，不建议在生产环境中使用。
-> - 切换代价模型版本可能会引起查询计划的变动。
+> 切换代价模型版本可能会引起查询计划的变动。
```
Comment (Oreoxmt) [inline system-variables.md:844]:
```suggestion
> - 自 v6.5.0 开始，新创建的 TiDB 集群默认使用 Cost Model Version 2。如果从 v6.4.0 及之前版本的集群升级到 v6.5.0 及之后的版本，`tidb_cost_model_version` 的值不发生变化。
> - 切换代价模型版本可能会引起查询计划的变动。
```
@qw4990 @chrysan PTAL

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: system-variables.md [hunk 9]
```diff
@@ -841,15 +841,15 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 > **警告：**
 >
-> - 当前 Cost Model Version 2 为实验特性，不建议在生产环境中使用。
+> - 自 v6.5.0 开始，新创建的 TiDB 集群默认使用 Cost Model Version 2。如果从 v6.4.0 及之前版本的集群升级到 v6.5.0 及之后的版本，`tidb_cost_model_version` 的值不发生变化。
```
Comment (TomShawn) [inline system-variables.md:844]: 这一句似乎算不上 warning？

File: system-variables.md [hunk 10]
```diff
@@ -841,15 +841,15 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 > **警告：**
```
Comment (Oreoxmt) [inline system-variables.md:842]:
```suggestion
> **注意：**
```

**Comment by TomShawn** [comment]
LGTM

File: cost-model.md [hunk 11]
```diff
@@ -40,13 +40,10 @@ mysql> SHOW CREATE TABLE t;
...
-
 TiDB v6.2.0 引入了新的代价模型 Cost Model Version 2。
 
 Cost Model Version 2 对代价公式进行了更精确的回归校准，调整了部分代价公式，比此前版本的代价公式更加准确。
 
-你可以通过设置变量 [`tidb_cost_model_version`](/system-variables.md#tidb_cost_model_version-从-v620-版本开始引入) 来控制代价模型的版本。
\ No newline at end of file
+你可以通过设置变量 [`tidb_cost_model_version`](/system-variables.md#tidb_cost_model_version-从-v620-版本开始引入) 来控制代价模型的版本。
+
+需要注意，切换代价模型版本可能引起执行计划的变动。
```
Comment (chrysan) [inline cost-model.md:49]: 这句可以 highlight 吗

Comment (Oreoxmt) [reply cost-model.md]: https://github.com/pingcap/docs-cn/pull/11825/commits/bab0121431dbbaedd9d6dca12e33482d279398cf Done

**Review by chrysan — APPROVED** [review]

File: system-variables.md [hunk 12]
```diff
@@ -839,17 +839,17 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
 - 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
-- 默认值：`1`
-- 可选值：`[1, 2]`
-- TiDB v6.2.0 引入了代价模型 [Cost Model Version 2](/cost-model.md#cost-model-version-2)，在内部测试中比此前版本的代价模型更加准确。
-- 通过将 `tidb_cost_model_version` 设置为 `2` 可以启用 Cost Model Version 2 代价模型，设置为 `1` 则继续使用 Cost Model Version 1 代价模型。
+- 默认值：`2`
+- 可选值：
+    - `1`：使用 Cost Model Version 1 代价模型。TiDB v6.4.0 及之前的版本默认使用 Cost Model Version 1。
+    - `2`：使用 Cost Model Version 2 代价模型。TiDB v6.2.0 引入了代价模型 [Cost Model Version 2](/cost-model.md#cost-model-version-2)，在内部测试中比 Version 1 版本的代价模型更加准确。
```
Comment (qw4990) [inline system-variables.md:852]:
```suggestion
    - `2`：使用 Cost Model Version 2 代价模型。TiDB v6.5.0 发布了代价模型 [Cost Model Version 2](/cost-model.md#cost-model-version-2)，在内部测试中比 Version 1 版本的代价模型更加准确。
```

Comment (Oreoxmt) [reply system-variables.md:852]: LGTM

**Pull request merged**